crates/storage/db/src/abstraction/cursor.rs

```diff
@@ -38,6 +38,16 @@ pub trait DbCursorRO<'tx, T: Table> {
     where
         Self: Sized;
 
+    /// Returns an iterator starting at a key greater or equal than `start_key` and ending at a key
+    /// less than `end_key`
+    fn walk_range<'cursor>(
+        &'cursor mut self,
+        start_key: T::Key,
+        end_key: T::Key,
```

Comment:
this should be a `Range` type

Reply:
follow it up w/ b701581295c5e5d1ac4316cc35ab96435cd32507